inst/examples/dashboard/style.scss

```
@@ -139,3 +250,133 @@ body {
 .ms-Shimmer-container {
   margin: 10px 0;
 }
+
+#app_header {
+        padding: 12px 10px;
+        justify-content: center;
+        align-items: center;
+      }
```

Please follow the indentation style already present in the file (2 spaces). Ditto for the rest of the file.

```suggestion
#app_header {
  padding: 12px 10px;
  justify-content: center;
  align-items: center;
}
```